Corona-Warn-App/src/main/java/de/rki/coronawarnapp/notification/TestResultAvailableNotificationService.kt

```diff
@@ -21,6 +23,11 @@ class TestResultAvailableNotificationService @Inject constructor(
     suspend fun showTestResultAvailableNotification(testResult: TestResult) {
         if (foregroundState.isInForeground.first()) return
 
+        if (!LocalData.isNotificationsTestEnabled) {
```

Comment:
This should be moved to the worker so all abort conditions are checked at the same spot